### ISSUE TITLE
Can't unserialize strings longer than 9 characters

### ISSUE THREAD

**Issue by 3onyc** [steelbrain/php-serialize#1]
String lengths of longer than 1 character get truncated to first character only, so for example a 17 character long string will be decoded as if it's 1 character.

```javascript
> var PHPSerialize = require("php-serialize");
> console.log(PHPSerialize.unserialize(PHPSerialize.serialize("stringlongerthan10characters")));
"s
```

**Comment by steelbrain** [comment]
This is very weird, I remember I used to work with very long objects with this in a production site

**Comment by steelbrain** [comment]
Found the problem, expect a fix soon

**Comment by steelbrain** [comment]
Fixed by https://github.com/steelbrain/php-serialize/pull/2

Released in 1.1.0
